request.js

```diff
@@ -778,6 +778,8 @@ Request.prototype.start = function () {
           var e = new Error('ESOCKETTIMEDOUT')
           e.code = 'ESOCKETTIMEDOUT'
           e.connect = false
+          e.host = self.host
+          e.port = self.port
```

Comment:
@FredKSchott any idea why `self.port` is kept as string after url parse? I had to cast the test to string.